src/cordova/apple/xcode/ios/PaperVPN.xcodeproj/project.pbxproj

```diff
@@ -1089,32 +1095,32 @@
 				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
 				CLANG_WARN_SUSPICIOUS_MOVE = YES;
 				CLANG_WARN_UNREACHABLE_CODE = YES;
-				CODE_SIGN_ENTITLEMENTS = "$(SRCROOT)/Outline/VpnExtension.entitlements";
+				CODE_SIGN_ENTITLEMENTS = "$(SRCROOT)/PaperVPN/VpnExtension.entitlements";
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				COPY_PHASE_STRIP = NO;
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
-				DEVELOPMENT_TEAM = QT8Z3Q9V3A;
+				DEVELOPMENT_TEAM = PZ6PPU9KKM;
```

Comment:
I think, we should join project files changes with https://github.com/paperpaperru/papervpn/tree/papervpn/id/apple_papervpn_v2